### PR TITLE
sidenav icon class update

### PR DESCRIPTION
**Pull request by diego-ruiz-rei**
## Description
We have a icon classes for success, error, pending (blank), but would like to add one for warning too

## Type of Change (Select One and Apply Github Label)

- [ ] Bug fix (non-breaking change which fixes an issue) -> Apply bugfix label
- [ ] New feature (non-breaking change which adds functionality) -> Apply enhancement label
- [ ] Breaking change (fix or feature that would cause existing functionality to change) -> Apply breaking label

## Which browsers have you tested?
- [ ] Internet Explorer 11
- [ ] Edge
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

## Checklist:

- [ ] I have read the (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md] document.
- [ ] My code passes the automated linter.
- [ ] This code has been reviewed by another team member and passes the reviewer checklist found in (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md]
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code is 508 compliant as tested by AMP and JAWS
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/ui-kit/components/sidenav/sidenav/sidenav.template.html [hunk 1]
```diff
@@ -30,6 +30,7 @@
       [ngClass]="{
         'active': isSelected($index),
         'completed': child.iconClass == 'completed',
+        'warning': child.iconClass == 'warning',
```
Comment (thomastighe) [inline src/ui-kit/components/sidenav/sidenav/sidenav.template.html]: Does this CSS already exist or is this custom CSS?

Comment (diego-ruiz-rei) [reply src/ui-kit/components/sidenav/sidenav/sidenav.template.html]: its in the sam-front-end css, don't see it in sam-ui-elements anywhere

Comment (thomastighe) [reply src/ui-kit/components/sidenav/sidenav/sidenav.template.html]: Is that where the rest of these classes are as well?

Comment (diego-ruiz-rei) [reply src/ui-kit/components/sidenav/sidenav/sidenav.template.html]: yep im seeing it the sam-front-end css as well, should those get added into sam-ui-elements css?

Comment (thomastighe) [reply src/ui-kit/components/sidenav/sidenav/sidenav.template.html]: Yeah that is a bigger deal.

**Comment by thomastighe** [comment]
Approve

**Pull request merged**